wpilib/src/pneumatics.rs

```diff
@@ -218,101 +244,93 @@ impl DoubleSolenoid {
     pub fn is_rev_blacklisted(&self) -> bool {
         self.reverse.is_blacklisted()
     }
-
-    pub fn module(&self) -> &SolenoidModule {
-        self.forward.module()
-    }
 }
 
+/// Represents a compressor hooked up to a PCM.
+/// Note that a Compressor object does not need to created for a compressor to function.
+/// A Compressor object is only used alter the default closed loop behavior.
 #[derive(Debug)]
 pub struct Compressor {
-    compressor_handle: HAL_CompressorHandle,
-    module: i32,
+    handle: HAL_CompressorHandle,
+}
+
+impl Default for Compressor {
+    #[inline]
+    /// Returns the `Compressor` on the default PCM.
+    fn default() -> Self {
+        Self::with_module(PneumaticsControlModule::DEFAULT)
+    }
 }
 
 impl Compressor {
-    #[allow(clippy::new_ret_no_self)]
-    pub fn new() -> HalResult<Self> {
-        Self::new_with_module(sensor_util::default_solenoid_module())
+    /// Returns the `Compressor` on the default PCM.
+    pub fn new() -> Self {
+        Self::default()
```

Comment:
It seems rather silly to have `new` delegate to a hand-written `Default` implementation...